pkg/asset/internal/templates.go

```diff
@@ -913,21 +913,71 @@ spec:
     - --name=boot-etcd
     - --listen-client-urls=http://0.0.0.0:12379
     - --listen-peer-urls=http://0.0.0.0:12380
-    - --advertise-client-urls=http://$(MY_POD_IP):12379
-    - --initial-advertise-peer-urls=http://$(MY_POD_IP):12380
-    - --initial-cluster=boot-etcd=http://$(MY_POD_IP):12380
+    - --advertise-client-urls=http://{{ .BootEtcdServiceIP }}:12379
```

Comment:
The self-hosted etcd cluster is now configured to migrate the boot etcd through its service IP instead of its actual (pod) IP. That way it doesn't have to look up the pod IP when doing the migration (which was kind of a hack)